Android wear support

votreglandeur opened the pull request:
Hello,

Thank for your nice job.
It will be nice if you can add a version for Android wear. Having wheel information on my Huawei watch will be perfect.

Thanks

Ask369 committed:
Update PebbleService.java

Fix of several errors in transferring details part data to Pebble.

Commit:
Merge pull request #5 from Ask369/patch-1

Update PebbleService.java

Commit:
fix Inmotion V8 speed, add new features

palachzzz committed:
Update README.md

palachzzz committed:
Update README.md

Commit:
FIX:
- Excessive logging, now the logs files 7-10 times smaller

Add:
- Wheel mode in second tab, and in log
- Model, Version and Serial number in second tab.

Commit:
Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid

Commit:
fix check calc, logging, add alert class and alert logging

Commit:
Wheel settings start

Commit:
fix + add

Commit:
added basic control functions

Commit:
bug fix, added speed and volume changing

Commit:
+alert

Commit:
remove alert

Commit:
log fix + fw ver fix

Commit:
text bug fix

Commit:
fix seekbar UI update + add alert

Commit:
pedals inmo + king fix

Commit:
fixed logging without locations, settings improvement

Commit:
fixed first start setting bug

Commit:
KS mode and version fix

Commit:
gotway temperature fixed

Commit:
KS negative current + disabled wheelsettings by default

Commit:
code clearing

Commit:
back click in preference fix

Commit:
fix back in menu

Commit:
+new

Commit:
base pref ks + gw

Commit:
-150 amps in alarm

juliomap committed:
Add support for Samsung Gear Tizen based watches

palachzzz committed:
Merge pull request #1 from juliomap/master

Add support for Samsung Gear Tizen based watches

Commit:
preferences, user trip, reset max speed, negative total in gw

Commit:
gw calibration, userdist logic changed, alarmtemp

Commit:
Merge branch 'improve_settings'

Commit:
merge

Commit:
log fix, avg speeds, ride time, calibration, user dist reset fix, inmo keepalive=200, inmo unknown alert fix, user dist multiwheel, ks light fix

Commit:
gw MCM, temp alarm fix

Commit:
avg riding fix, and rename

Commit:
alarm popup, fix avg riding, moved gw MCM

Commit:
gw-84, inmo-zerospeed, reset for use_ratio

Commit:
gw 84 fix, thanks @MacPara (https://github.com/MacPara)

Commit:
+minor fix

Commit:
gradle 4.4 updates to the latest api

Commit:
Inmotion V10 support

Commit:
manual mac address

Commit:
gotway-100v, ks-18l_battery_meter, v10f, inmotion_45kmh

Commit:
version fix

Commit:
some improvement

Commit:
ks18l - battery meter fix

Commit:
inmo debug

Commit:
log naming fix

Commit:
I fixed very old inmotion bug ))

Commit:
Inmotion connection fix, auto speed of connection, new battery meter for all wheels, now it is realy stop after close. V10F distance fix

Commit:
start

Commit:
build number

Commit:
comments

Commit:
some adds

Commit:
ninebot support alfa

Commit:
futher work about ninebot_z support

Commit:
fixing some bugs

Commit:
beta release

Commit:
Create LICENSE

Commit:
ninebot end + rw

Commit:
glide3, gps speed, heading

Commit:
small fix

Commit:
roll back percents

Commit:
+ License

Commit:
shutdown fix for drive.google.com

dotarsoja committed:
add KS settings

dotarsoja committed:
Add Speed Alarms settings

dotarsoja committed:
add max_speed request

dotarsoja committed:
add answer for request

dotarsoja committed:
edit settings view

dotarsoja committed:
load settings speed&alerts

dotarsoja committed:
started work on localization

dotarsoja committed:
mid point commit

dotarsoja committed:
finish KingSong localization

dotarsoja committed:
finish gotway localization

dotarsoja committed:
samll fix

dotarsoja committed:
finish inmotion localization

dotarsoja committed:
finish ninebot_z localization

dotarsoja committed:
finish localization

Commit:
Merge branch 'master' into master

dotarsoja committed:
fix grammar and typos

Commit:
Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid

palachzzz committed:
Merge pull request #7 from dotarsoja/master

Add Russian localisation and kingsong alarms preferences.

Commit:
update gradle

Commit:
Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid

Commit:
gw fix
lang fix

hyperair committed:
Fix usage of carriage returns in csv exports to google drive

hyperair committed:
Fix rockwheel detection

mBtName isn't populated (is an empty string) if we don't start from a fresh
bluetooth pairing with the GT16, so the battery gauge is wrongly calculated
based on a 67.2V battery instead.

Instead, check mName for a "ROCKW" prefix, as this is available even on
subsequent sessions.

palachzzz committed:
Merge pull request #8 from hyperair/gdrive-line-endings

Fix usage of carriage returns in csv exports to google drive

palachzzz committed:
Merge pull request #9 from hyperair/rockwheel-battery-estimation

Fix rockwheel detection

Commit:
beeps + alarms

Commit:
Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid

Commit:
fix

juliomap commented:
I am afraid that Android wear has nothing to do with Tizen.
But in the forum I think I have seen somebody installing the Android version successfuly in Android wear. The only issue being the small size of the screen.
Anyway this is something you shoukd ask to the android developers who would work in a more similar environment.

Enviado desde Yahoo Mail con Android 
 
  El vie., jul. 5, 2019 a 13:38, votreglandeur<notifications@github.com> escribió:   
Hello,

Thank for your nice job.
It will be nice if you can add a version for Android wear. Having wheel information on my Huawei watch will be perfect.

Thanks

You can view, comment on, or merge this pull request online at:

  https://github.com/cedbossneo/WheelLogAndroid/pull/1

Commit Summary
   
   - Update PebbleService.java
   - Merge pull request #5 from Ask369/patch-1
   - fix Inmotion V8 speed, add new features
   - Update README.md
   - Update README.md
   - FIX:
   - Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid
   - fix check calc, logging, add alert class and alert logging
   - Wheel settings start
   - fix + add
   - added basic control functions
   - bug fix, added speed and volume changing
   - +alert
   - remove alert
   - log fix + fw ver fix
   - text bug fix
   - fix seekbar UI update + add alert
   - pedals inmo + king fix
   - fixed logging without locations, settings improvement
   - fixed first start setting bug
   - KS mode and version fix
   - gotway temperature fixed
   - KS negative current + disabled wheelsettings by default
   - code clearing
   - back click in preference fix
   - fix back in menu
   - +new
   - base pref ks + gw
   - -150 amps in alarm
   - Add support for Samsung Gear Tizen based watches
   - Merge pull request #1 from juliomap/master
   - preferences, user trip, reset max speed, negative total in gw
   - gw calibration, userdist logic changed, alarmtemp
   - Merge branch 'improve_settings'
   - merge
   - log fix, avg speeds, ride time, calibration, user dist reset fix, inmo keepalive=200, inmo unknown alert fix, user dist multiwheel, ks light fix
   - gw MCM, temp alarm fix
   - avg riding fix, and rename
   - alarm popup, fix avg riding, moved gw MCM
   - gw-84, inmo-zerospeed, reset for use_ratio
   - gw 84 fix, thanks @MacPara (https://github.com/MacPara)
   - +minor fix
   - gradle 4.4 updates to the latest api
   - Inmotion V10 support
   - manual mac address
   - gotway-100v, ks-18l_battery_meter, v10f, inmotion_45kmh
   - version fix
   - some improvement
   - ks18l - battery meter fix
   - inmo debug
   - log naming fix
   - I fixed very old inmotion bug ))
   - Inmotion connection fix, auto speed of connection, new battery meter for all wheels, now it is realy stop after close. V10F distance fix
   - start
   - build number
   - comments
   - some adds
   - ninebot support alfa
   - futher work about ninebot_z support
   - fixing some bugs
   - beta release
   - Create LICENSE
   - ninebot end + rw
   - glide3, gps speed, heading
   - small fix
   - roll back percents
   - + License
   - shutdown fix for drive.google.com
   - add KS settings
   - Add Speed Alarms settings
   - add max_speed request
   - add answer for request
   - edit settings view
   - load settings speed&alerts
   - started work on localization
   - mid point commit
   - finish KingSong localization
   - finish gotway localization
   - samll fix
   - finish inmotion localization
   - finish ninebot_z localization
   - finish localization
   - Merge branch 'master' into master
   - fix grammar and typos
   - Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid
   - Merge pull request #7 from dotarsoja/master
   - update gradle
   - Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid
   - gw fix
   - Fix usage of carriage returns in csv exports to google drive
   - Fix rockwheel detection
   - Merge pull request #8 from hyperair/gdrive-line-endings
   - Merge pull request #9 from hyperair/rockwheel-battery-estimation
   - beeps + alarms
   - Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid
   - fix

File Changes
   
   -  A LICENSE (675) 
   -  M README.md (9) 
   -  M app/build.gradle (41) 
   -  A app/libs/accessory-v2.5.1.jar (0) 
   -  A app/libs/sdk-v1.0.0.jar (0) 
   -  A app/release/output.json (1) 
   -  M app/src/main/AndroidManifest.xml (26) 
   -  M app/src/main/java/com/cooper/wheellog/BluetoothLeService.java (91) 
   -  A app/src/main/java/com/cooper/wheellog/FileLoggingTree.java (61) 
   -  A app/src/main/java/com/cooper/wheellog/GearSAPServiceProviderConnection.java (42) 
   -  A app/src/main/java/com/cooper/wheellog/GearService.java (284) 
   -  M app/src/main/java/com/cooper/wheellog/GoogleDriveService.java (2) 
   -  M app/src/main/java/com/cooper/wheellog/LoggingService.java (42) 
   -  M app/src/main/java/com/cooper/wheellog/MainActivity.java (268) 
   -  M app/src/main/java/com/cooper/wheellog/PebbleService.java (10) 
   -  M app/src/main/java/com/cooper/wheellog/PreferencesFragment.java (358) 
   -  M app/src/main/java/com/cooper/wheellog/ScanActivity.java (9) 
   -  M app/src/main/java/com/cooper/wheellog/WheelData.java (770) 
   -  M app/src/main/java/com/cooper/wheellog/WheelLog.java (7) 
   -  M app/src/main/java/com/cooper/wheellog/utils/Constants.java (21) 
   -  M app/src/main/java/com/cooper/wheellog/utils/InMotionAdapter.java (583) 
   -  A app/src/main/java/com/cooper/wheellog/utils/NinebotZAdapter.java (757) 
   -  M app/src/main/java/com/cooper/wheellog/utils/NotificationUtil.java (4) 
   -  M app/src/main/java/com/cooper/wheellog/utils/SettingsUtil.java (14) 
   -  M app/src/main/java/com/cooper/wheellog/views/WheelView.java (38) 
   -  M app/src/main/res/layout/main_view_three.xml (4) 
   -  M app/src/main/res/layout/main_view_two.xml (88) 
   -  A app/src/main/res/values-ru-rRU/strings.xml (145) 
   -  M app/src/main/res/values/arrays.xml (83) 
   -  A app/src/main/res/values/services_ninebot_z.xml (30) 
   -  M app/src/main/res/values/strings.xml (159) 
   -  M app/src/main/res/values/wheel_types.xml (2) 
   -  M app/src/main/res/xml/preferences.xml (33) 
   -  M app/src/main/res/xml/preferences_alarms.xml (76) 
   -  A app/src/main/res/xml/preferences_gotway.xml (67) 
   -  A app/src/main/res/xml/preferences_inmotion.xml (55) 
   -  A app/src/main/res/xml/preferences_kingsong.xml (87) 
   -  M app/src/main/res/xml/preferences_logs.xml (16) 
   -  A app/src/main/res/xml/preferences_ninebot_z.xml (11) 
   -  M app/src/main/res/xml/preferences_speed.xml (10) 
   -  A app/src/main/res/xml/preferences_trip.xml (15) 
   -  M app/src/main/res/xml/preferences_watch.xml (6) 
   -  A app/src/main/res/xml/sapservices.xml (50) 
   -  M build.gradle (3) 
   -  D gradlew (160) 
   -  M gradlew.bat (174) 

Patch Links:
   
   - https://github.com/cedbossneo/WheelLogAndroid/pull/1.patch
   - https://github.com/cedbossneo/WheelLogAndroid/pull/1.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.

marccardinal committed:
Bringing in the garmin connectiq lib

marccardinal committed:
Initial commit of the GarminConnectIQ service

marccardinal committed:
Start the GarminConnectIQ service at the same time as pebble

marccardinal committed:
Link to the watch app and source

marccardinal committed:
Update README.md

marccardinal committed:
Update README.md

nicolas2k committed:
Update README.md

marccardinal committed:
Merge pull request #1 from nicolas2k/patch-1

Update README.md

palachzzz committed:
Merge pull request #10 from marccardinal/garmin-connect-iq

Introducing support for Garmin Connect IQ devices

Commit:
+chineese lang pack

marccardinal committed:
Temporarily disabled the android to watch push of data and introduced nanohttpd

Details about the issue can be found here:
https://forums.garmin.com/developer/connect-iq/f/legacy-bug-reports/5144/failure_during_transfer-issue-again-now-using-comm-sample
People have reported receiving a FAILURE_DURING_TRANSFER when sending messages to the watch with a short interval, this also
seems to happen after a certain number of messages have been sent to the watch according to people on the forum. On my end
two people one with a F5X+ and another with a VivoActive 3 Music have reporting the issue intermittently too. So instead of
having the android app push the data I am now having the watch poll the data via a /data request to localhost on an ephemeral
port. This is a workaround until Garmin decides to tackle the underlying issue (the forum post is from 2yrs+ ago)

marccardinal committed:
Moving nanohttpd to localhost + ephemeral port

Commit:
+ Garmin, force English,

Commit:
Seba's fork languages port

Commit:
- market

marccardinal committed:
Added a featureflag for nanohttpd method requiring garmin connect 4.22 (not out yet)

marccardinal committed:
Garmin Connect v4.22 is now out, enabling nanohttpd

marccardinal committed:
GCIQ whitelists 127.0.0.1, not localhost

palachzzz committed:
Merge pull request #11 from marccardinal/garmin-connect-iq

Fixing the intermittent FAILURE_DURING_TRANSFER

Commit:
+split screen support, + ninebot e support

Commit:
Merge branch 'master' of https://github.com/palachzzz/WheelLogAndroid

Commit:
- minus debug

Commit:
+ android 7 and below compatibility fix
